repos/system_upgrade/el7toel8/actors/checkfstaboptions/tests/test_checkfstaboptions.py

```diff
@@ -0,0 +1,89 @@
+from leapp.models import FstabEntry, StorageInfo
+from leapp.snactor.fixture import current_actor_context
+from leapp.reporting import Report
+
+
+MOUNTS_CORRECT_1 = [
+    FstabEntry(
+        fs_spec='/dev/mapper/rhel_ibm--p8--kvm--03--guest--02-root',
+        fs_file='/',
+        fs_vfstype='xfs',
+        fs_mntops='defaults',
+        fs_freq='0',
+        fs_passno='0'
+    ),
+    FstabEntry(
+        fs_spec='UUID=c3ae2d1a-8088-45c2-9482-4aacb2c075cf',
+        fs_file='/boot',
+        fs_vfstype='xfs',
+        fs_mntops='defaults',
+        fs_freq='0',
+        fs_passno='0'
+    )
+]
+
+MOUNTS_INVALID = [
+    FstabEntry(
+        fs_spec='/dev/mapper/rhel_ibm--p8--kvm--03--guest--03-root',
+        fs_file='/',
+        fs_vfstype='xfs',
+        fs_mntops='defaults,nobarrier',
+        fs_freq='0',
+        fs_passno='0'
+    ),
+    FstabEntry(
+        fs_spec='UUID=c3ae2d1a-8088-45c2-9482-4aacb2c075cf',
+        fs_file='/boot',
+        fs_vfstype='xfs',
+        fs_mntops='defaults,barrier,noatime,delaylog,ihashsize',
+        fs_freq='0',
+        fs_passno='0'
+    )
+]
+
+MOUNTS_CORRECT_2 = [
+    FstabEntry(
+        fs_spec='/dev/mapper/luks-afe3592c-42c7-4702-bb1f-d403f63034c3',
+        fs_file='/',
+        fs_vfstype='ext4',
+        fs_mntops='defaults,x-systemd.device-timeout=0',
+        fs_freq='1',
+        fs_passno='1'
+    ),
+    FstabEntry(
+        fs_spec='UUID=df14b440-a7cd-4af0-aa4e-bfb6216813d9',
+        fs_file='/boot',
+        fs_vfstype='ext4',
+        fs_mntops='defaults',
+        fs_freq='1',
+        fs_passno='2'
+    )
+]
+
+
+def test_ok(current_actor_context):
```

Comment:
it is might be a good idea to add docstrings (or comments) to your tests in order to explain what do you actually testing

Reply:
Basically fixed this one, in the current tests it's more obvious.